Assets/Scripts/Audio/AudioUtils.cs

```diff
@@ -50,7 +50,7 @@ public static GameObject Play2DAudioStream(MP3StreamReader audioStream)
 
 		var audioSource = gameObject.AddComponent<AudioSource>();
 		//audioSource.clip = CreateStreamingAudioClip("tmpAudioClip", audioStream);
-		//audioSource.loop = true;
+		audioSource.loop = true;
```

Comment:
So, long story short, I ended up implementing a hack to get runtime MP3 streaming working. I'm using OnAudioFilterRead to play the music right now, and the only reason I'm adding an AudioSource is because that enables OnAudioFilterRead IIRC. So setting audioSource.loop = true here won't have any effect. It will have an effect when I fix the audio hacks, though, so I'll leave this change in. Just FYI.